frontend/src/components/pages/CreateReview/BookModal.tsx

```diff
@@ -0,0 +1,275 @@
+        <ModalBody>
+          <Grid templateColumns="repeat(2, 1fr)" gap={10}>
+            <GridItem>
+              <Stack spacing={4}>
+                <Stack w="100%" spacing={5} isInline>
+                  <AddStringInput
+                    id="titlePrefix"
+                    label="Prefix"
+                    name="title-prefix"
+                    required
+                    maxWidth="40%"
+                    inputField={titlePrefix}
+                    setInputField={setTitlePrefix}
+                  />
+                  <AddStringInput
+                    id="title"
+                    label="Title"
+                    name="title"
+                    required
+                    inputField={title}
+                    setInputField={setTitle}
+                  />
+                </Stack>
+                <Stack align="center" spacing={5} isInline>
+                  <AddStringInput
+                    id="seriesName"
+                    label="Series"
+                    name="series-name"
+                    required={false}
+                    inputField={seriesName}
+                    setInputField={setSeriesName}
+                  />
+                  <AddStringInput
+                    id="seriesOrder"
+                    label="Book number"
+                    name="series-order"
+                    required={false}
+                    maxWidth="40%"
+                    inputField={seriesOrder}
+                    setInputField={setSeriesOrder}
+                  />
+                </Stack>
+                <AddStringInputList
+                  id="authors"
+                  label="Author"
+                  name="author-name"
+                  required
+                  inputFields={authors}
+                  setInputFields={setAuthors}
+                />
+                <AddStringInputList
+                  id="illustrator"
+                  label="Illustrator"
+                  name="illustrator-name"
+                  required={false}
+                  inputFields={illustrators}
+                  setInputFields={setIllustrator}
+                />
+                <AddStringInputList
+                  id="translator"
+                  label="Translator"
+                  name="translator-name"
+                  required={false}
+                  inputFields={translators}
+                  setInputFields={setTranslator}
+                />
+                <Stack align="center" spacing={5} isInline>
+                  <AddStringInput
+                    id="publishers"
+                    label="Publisher"
+                    name="publisher-name"
+                    required
+                    inputField={publisher}
+                    setInputField={setPublisher}
+                  />
+                  <FormControl id="Publication Year" isRequired>
+                    <FormLabel mb={2}>Publication Year</FormLabel>
+                    <AddNumberInput
```

Comment:
The designs have this as a drop down. Not sure what the values would be in that drop down. Maybe sync with the designers to make sure everyone is on the same page here?

Reply:
Yea sounds good, I thought having a drop down was excessive but ill double check!